release notes/v0.54.0.md

```diff
@@ -0,0 +1,304 @@
+k6 `v0.54.0` is here 🎉! This release includes:
+
+- A new experimental CSV module
+- New ECMAScript features
+- Updated logo and branding
+
+## Breaking changes
+
+- [#3913](https://github.com/grafana/k6/pull/3913) changes the mapping of Golang's `math/big.Int` type to `bigint` type in k6.
+- [#3922](https://github.com/grafana/k6/pull/3922) removes `lib.Min` and `lib.Max` from k6's Go API, which could affect custom extensions that rely on these functions.
+- [#3838](https://github.com/grafana/k6/pull/3838) removes `k6/experimental/timers` - they are now available globally and no import is needed.
+- [#3944](https://github.com/grafana/k6/pull/3844) updates to `k6/experimental/websockets` made the binaryType default value equal to "blob". With this change, `k6/experimental/websockets` is now compliant with the specification.
+
+## New features
+
+### Branding changes and logo [`#3946`](https://github.com/grafana/k6/pull/3946), [`#3953`](https://github.com/grafana/k6/pull/3953)
+
+As part of joining Grafana Labs in 2021, k6 was renamed to Grafana k6. The original k6 logo and branding was purple, which didn't fit very well next to the Grafana Labs orange logo and all its other products.
+
+In this release, we have a new logo in a new color, and the terminal banner has been redesigned to match the current branding more closely.
+
+<p align="center">
+  <img src="https://github.com/grafana/k6/blob/31e3db711850fdd270e56e53feb3a2e4209e542a/assets/logo.svg" alt="Grafana k6 logo" width="210">
+</p>
+
+### New `k6 cloud run --local-execution` flag for local execution of cloud tests [`#3904`](https://github.com/grafana/k6/pull/3904), and [#3931](https://github.com/grafana/k6/pull/3931)
```

Comment:
Very small nit, but maybe we can move this below the CSV heading, so both cloud command news are listed next to each other?

Reply:
Done